vars/clearlinux.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Most `vars/` files (apart from `vars/main.yml`) can be removed since you included `tasks/selinux.yml`